recipe/meta.yaml

```
@@ -1,36 +1,46 @@
-{% set name = "universal_pathlib" %}
 {% set version = "0.2.6" %}
 
 package:
-  name: {{ name|lower }}
+  name: universal_pathlib
   version: {{ version }}
 
 source:
   # If getting the source from GitHub, remove the line above,
   # uncomment the line below, and modify as needed. Use releases if available:
-  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://pypi.org/packages/source/u/universal_pathlib/universal_pathlib-{{ version }}.tar.gz
   sha256: 50817aaeaa9f4163cb1e76f5bdf84207fa05ce728b23fd779479b3462e5430ac
 
 build:
   noarch: python
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
 
-requirements:
-  host:
-    - pip
-    - python {{ python_min }}
-    - setuptools >=48
-    - setuptools-scm >=6.3.1
-  run:
-    - python >={{ python_min }}
-    - fsspec >=2022.1.0,!=2024.3.1
+outputs:
+  - name: universal_pathlib
+    script: build_base.sh
```

Like [in this example]( https://github.com/conda-forge/astropy-feedstock/blob/97e2f2cd0db98412165f25bdf2ecb2862ecbbcb6/recipe/meta.yaml#L87-L90 ), we can still keep the `script` in the recipe

```suggestion
    build:
      script: {{ PYTHON }} -m pip install . -vv
```